tests/baselines/reference/abstractPropertyInConstructor.errors.txt

```diff
@@ -34,6 +35,11 @@ tests/cases/compiler/abstractPropertyInConstructor.ts(9,14): error TS2715: Abstr
     
         abstract method(num: number): void;
     
+        other = this.prop;
+                     ~~~~
+!!! error TS2715: Abstract property 'prop' in class 'AbstractClass' cannot be accessed in the constructor.

```

Comment:
should I update the error message to something like `Abstract property 'prop' in class 'AbstractClass' cannot be accessed during class initialization.`?

Reply:
It's technically running in the constructor...